types/migrations/20220714123910_edited_collections_2.sql

```diff
@@ -0,0 +1,6 @@
+-- Add migration script here
+create type chapter_t as enum ('intro', 'body');
+
+alter table collection_chapter
+    add chapter_type chapter_t;
```

Comment:
Types are allowed to have the same name as columns, as far as I know. So, we could have `chapter_type` for both. I think we could use a more descriptive name here though since "chapter type" could mean a lot of things, plus we ideally don't need `chapter_` at the beginning of the column name (`collection_chapter.chapter_type` is a bit redundant). Perhaps simply `section`, with type name `collection_section`? What do you think?